setup.py

```diff
@@ -56,8 +56,7 @@
     'aodncore>=0.31.0',
     'cc-plugin-imos>=1.3.0',
     'matplotlib==1.5.1',
-    'aodntools>=0.4.3',
-    'pandas==0.24.2'
```

Comment:
We may not need to pin the version, but we _are_ using pandas in several handlers, so we should keep it here.

Reply:
I'm trying to find documentation of the best practice approach for this (importing dependencies of dependencies), but removing it is in line with several other key packages, namely `compliance-checker` (installed by aodncore), `numpy` (installed by `compliance-checker`, so 2 "hops" away in the dependency chain) and `netCDF4` (installed by `compliance-checker`).

So basically all of the scientific libraries which are installed indirectly by aodncore, aodntools and cc-plugin-imos. Dependency clashes are a huge pain, so I was hoping for the simplest approach here. If we *know* that aodntools and aodncore install the scientific libs, I can't think of a good reason to explicitly duplicate the dependency here?

Reply:
My thinking was that we should have `pandas` listed here because it is explicitly imported by some of the handlers, unlike `compliance-checker`, which is only used indirectly via `aodncore`.
But `numpy` and `netCDF4` are also imported directly and not listed here as a dependency, which is ok.

So I agree, let's try to avoid dependency clashes.